src/highlight/HighlightList.tsx

```diff
@@ -23,15 +27,18 @@ export function HighlightList({ annotations, className, onSelect }: Props): JSX.
     useOutsideEvent('mouseup', rootElRef, (): void => setIsListening(true));
 
     return (
-        <svg
-            ref={rootElRef}
-            className={classNames(className, 'ba-HighlightList', { 'is-listening': isListening })}
-            data-resin-component="highlightList"
-        >
-            {annotations.map(({ id, target }) => (
-                <HighlightAnnotation key={id} annotationId={id} onSelect={onSelect} rects={target.shapes} />
-            ))}
-        </svg>
+        <>
+            <HighlightCanvas activeId={activeId} annotations={sortedAnnotations} />
+            <HighlightSvg
```

Comment:
Should these be HighlightTarget components? They'll need to be clickable, correct?